modules/conflict_resolver/jsx/conflictResolverIndex.js

```diff
@@ -1,310 +0,0 @@
-import React, {Component} from 'react';
```

Comment:
The file `modules/conflict_resolver/jsx/.fix_conflict_form.js.swp` above (that I can't comment on..) shouldn't be included

Reply:
Resolved!